src/lib/containers/entity_finder/SelectionPane.tsx

```diff
@@ -16,23 +20,41 @@ export const SelectionPane: React.FC<SelectionPaneProps> = ({
   selectedEntities,
   toggleSelection,
 }: SelectionPaneProps) => {
+  if (typeof title === 'function') {
+    title = title(selectedEntities.size)
+  }
```

Comment:
e.g. for datasets this prop will be `(count) => count + " item" + count > 1 ? "s" : "" + " selected"`